docs/source/api.rst

```diff
@@ -40,13 +40,6 @@ Betweenness Centrality
     :members:
     :undoc-members:
 
-Edge Betweenness Centrality
```

Comment:
Don't we need that?

Reply:
We do, but since both Vertex and Edge are in the same file, the doc tool produces a bad output.  https://docs.rapids.ai/api/cugraph/nightly/api.html#centrality

The API is still in the docs with this change, and is searchable, there is just not an entry in the TOC.  I need to determine how to fix that, but it might involve refactoring the files